master/static/srv/entrypoint.sh

```diff
@@ -46,13 +45,14 @@ fi
 # training, so we try to query the user system for a valid HOME, or default to
 # the working directory otherwise.
 if [ "$HOME" = "/" ] ; then
-    HOME="$(set -o pipefail; getent passwd "$(whoami)" | cut -d: -f6)" || HOME="$WORKING_DIR"
+    HOME="$(set -o pipefail; getent passwd "$(whoami)" | cut -d: -f6)" || HOME=$PWD
```

Comment:
sorry to send it back one more time... but you need to use quotes when dereferencing a variable.